client/src/components/AlumniDirectory.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { Card, Image, Search, Pagination, Grid, Segment } from 'semantic-ui-react'
+
+/*
+props:
+- entries: list of alumni profiles
+*/
+export default class AlumniDirectory extends Component {
+    state = {
+        activePage: 1,
+        numEntries: Object.keys(this.props.entries).length,
```

Comment:
Any reason we're calling length on keys instead of the entries array itself?

Reply:
Not anymore - it used to be necessary because entries was a js object, but I switched over to a list.
I'll change it now